### PR TITLE
GEOMESA-1308 Added a deletefeatures command to the geomesa command li…

### PR DESCRIPTION
**Pull request by Afterglow375**
…ne tools

Signed-off-by: Alex Tatusko <atatusko@ccri.com>

### CODE REVIEW

**Comment by jahhulbert-ccri** [comment]
can we add a unit test like this: https://github.com/locationtech/geomesa/blob/master/geomesa-tools/src/test/scala/org/locationtech/geomesa/tools/accumulo/commands/IngestCommandTest.scala#L77 we haven't in the past but we should start invoking unit tests on these commands.

**Comment by jahhulbert-ccri** [comment]
need to update the geomesa-tools README.md and the corresponding docs RST file for command line tools

**Comment by jnh5y** [comment]
@Afterglow375 to split the difference, let's go for documentation in this PR, and unit tests another day.

File: geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/DeleteFeaturesCommand.scala [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+class DeleteFeaturesCommand(parent: JCommander) extends CommandWithKDS(parent) with LazyLogging {
+  override val command = "deletefeatures"
+  override val params = new DeleteFeaturesParameters
+
+  override def execute() = {
+    val sftName = params.featureName
+    val filter = Option(params.cqlFilter).map(ECQL.toFilter).getOrElse(Filter.INCLUDE)
+
+    logger.info(s"Deleting features from $sftName with filter $filter. This may take a few moments...")
+    ds.getFeatureSource(sftName).asInstanceOf[SimpleFeatureStore].removeFeatures(filter)
```
Comment (jnh5y) [inline geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/kafka/commands/DeleteFeaturesCommand.scala]: Did you have a chance to test this?  

I just realized that the command wouldn't know what Delete GeoMessage sends to send to Kafka...

Sorry that I didn't realize that earlier.

**Comment by Afterglow375** [comment]
I removed the KDS deletefeatures command for now due to this bug:

https://geomesa.atlassian.net/browse/GEOMESA-1312

**Pull request merged**